pcp/PCPrecord_actions.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+# Executed by systemd service 'PCPrecord.service'
+# See: /etc/systemd/system/PCPrecord.service
+################################################################
+
+# GLOBALS ###################
+# Include the PCP Functions file
+source $PWD/pcp_functions.inc
+
+FIFO="/tmp/pcpFIFO"                 # get from cmdline
+sample_rate=5                       # hardcode DEFAULT for now
+pmlogger_running="false"            # Initialize service as OFF
+om_workload_file="/tmp/openmetrics_workload.txt"
+om_workload_file_reset="/tmp/openmetrics_workload_reset.txt"
+
+#############################
+# Functions #################
+
+reset_om_metrics() {
+    echo "Reset OpenMetrics values"
+    cp ${om_workload_file_reset} ${om_workload_file}
+}
+
+error_exit() {
+    if [ "$?" != "0" ]; then
```

Comment:
Please place brace on it is own line
error_exit()
{
   < text>
}
Makes it easier to find the functions.
Should be done for all functions adding.